### PR TITLE
Add Action Syntax - Resize.MinimumPad

### PR DESCRIPTION
**Pull request by patrick-tolosa**

### CODE REVIEW

File: src/actions/resize/minimumPad.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+/**
+ * @implements IAction
+ */
+class MinimumPad {
+  /**
+   *
+   * @param {number} padWidth
+   * @param {number} padHeight
+   */
+  constructor(padWidth, padHeight) {
```
Comment (maoznir) [inline src/actions/resize/minimumPad.js:12]: Why not inherit from some general Resize class?

Comment (maoznir) [reply src/actions/resize/minimumPad.js:12]: Or implement a resize action interface?

Comment (patrick-tolosa) [reply src/actions/resize/minimumPad.js:12]: At this moment we don't have all the information of how everything looks like.
I think once we'll have more actions it'll be easier to consolidate.

There's no other Resize action here, so there's nothing to consolidate yet.

Comment (patrick-tolosa) [reply src/actions/resize/minimumPad.js:12]: Merging for now, we'll discuss reconsolidation once we have more code.

**Pull request merged**